open-craft/sprints
# [BB-1625] Rework retrieving accounts

This greatly improves the accounts' retrieval performance and the reliability of the API by moving to the different Tempo API endpoint and making cell-related calculations being performed directly on frontend.

For gathering data from a year the difference for me was 1.5m -> 17s.

## Testing instructions:
1. Set `.env` file (you can take find in a comment on the ticket).
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create frontend/.env.local and set REACT_APP_GOOGLE_CLIENT_ID there.
1. Load the fixtures (attached to the ticket) with `docker-compose -f local.yml exec django python manage.py loaddata fixtures_sprints.json`.
1. Navigate to frontend and run `npm i && npm start`.
1. Log in.
1. Check the Sustainability and Budget dashboard for overall/cells/yourself and see if the accounts look fine.
